be/src/olap/compaction.cpp

```diff
@@ -1223,8 +1223,10 @@ Status CloudCompactionMixin::construct_output_rowset_writer(RowsetWriterContext&
     ctx.write_type = DataWriteType::TYPE_COMPACTION;
 
     auto compaction_policy = _tablet->tablet_meta()->compaction_policy();
-    ctx.compaction_level =
-            _engine.cumu_compaction_policy(compaction_policy)->new_compaction_level(_input_rowsets);
+    if (_tablet->tablet_meta()->time_series_compaction_level_threshold() >= 2) {
+        ctx.compaction_level = _engine.cumu_compaction_policy(compaction_policy)
```

Comment:
What problem this code try to fix?

Reply:
If the compaction level is changed from 1 to 2,  be restarts, it will continuously merge the rowsets that have already been merged。

Reply:
If the compaction level is changed from one to two, be restarts, it will continuously merge the rowsets that have been merged.